client/src/components/StoreCard/StoreCard.test.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import StoreCard from "./StoreCard";
+import { Store } from "src/interfaces";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("StoreCard Component Tests", () => {
+  const TEST_STORE: Store = {
+    name: "Store1",
+    "merchant-id": "TEST",
+    "store-id": "TEST_STORE1",
+    latitude: -1,
+    longitude: -1,
+  };
+
+  let redirectedUrl = "";
```

Comment:
nit: instead of changing this in beforeEach, move to local variable as it might not be used/needed in all tests, so it is not required to make it a common test var

Reply:
Good point, I think the string is not necessary, replaced with a mocked function and changed test to expect `toBeCalledWith` instead of checking string matching.